packages/preview/scrutinize/0.2.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "scrutinize"
+version = "0.2.0"
+entrypoint = "src/lib.typ"
+authors = [
+  "Clemens Koza <https://github.com/SillyFreak/>"
+]
+license = "MIT"
+description = "A library for building exams, tests, etc. with Typst"
+repository = "https://github.com/SillyFreak/typst-packages/tree/main/scrutinize"
+keywords = ["exam", "test", "school", "teaching"]
+compiler = "0.11.0"
```

Comment:
You might want to use the opportunitity to add [`categories`](https://github.com/typst/packages/blob/main/CATEGORIES.md) and possibly also [`disciplines`](https://github.com/typst/packages/blob/main/DISCIPLINES.md).